tx-pool/src/component/pool_map.rs

```diff
@@ -329,6 +329,33 @@ impl PoolMap {
         conflicts
     }
 
+    pub(crate) fn estimate_fee_rate(
+        &self,
+        mut target_blocks: usize,
+        max_block_bytes: usize,
+        max_block_cycles: Cycle,
+        min_fee_rate: FeeRate,
+    ) -> FeeRate {
+        debug_assert!(target_blocks > 0);
+        let iter = self.entries.iter_by_score().rev();
+        let mut current_block_bytes = 0;
+        let mut current_block_cycles = 0;
+        for entry in iter {
+            current_block_bytes += entry.inner.size;
+            current_block_cycles += entry.inner.cycles;
+            if current_block_bytes >= max_block_bytes || current_block_cycles >= max_block_cycles {
+                target_blocks -= 1;
+                if target_blocks == 0 {
+                    return entry.inner.fee_rate();
+                }
+                current_block_bytes = entry.inner.size;
+                current_block_cycles = entry.inner.cycles;
+            }
+        }
+
+        min_fee_rate
```

Comment:
what is `min_fee_rate` used for in this function? seems it is not referenced in this function.

Reply:
this fn will return `min_fee_rate` when estimation target is not found.